dashboard/src/styles/main.css

```diff
@@ -5,6 +5,13 @@ body
     color: white;
 }
 
+hr {
```

Comment:
consider using  module based css instead of one giant stylesheet to avoid selector collisions/scope css to only the components that need them https://create-react-app.dev/docs/adding-a-css-modules-stylesheet/